### ISSUE TITLE
qwen3-8b-base 继续预训练时loss来回震荡

### ISSUE THREAD

**Issue by yyyhhl** [modelscope/ms-swift#6828]
训练命令：torchrun --standalone --nnodes=1 --node_rank=0 --nproc_per_node=4 --master_port=45391 ./swift_pt.py --model ./workdir/base_model --model_type qwen3 --dataset ./workdir/dataset/shuffled_dataset.jsonl --output_dir ./workdir/swift_output --attn_impl flash_attn --num_train_epochs 3 --torch_dtype bfloat16 --max_length 65536 --per_device_train_batch_size 1 --per_device_eval_batch_size 1 --sequence_parallel_size 1 --learning_rate 1e-5 --warmup_ratio 0.05 --weight_decay 1e-4 --max_grad_norm 1.0 --gradient_accumulation_steps 4 --save_strategy epoch --save_total_limit 3 --train_type full --deepspeed zero3_offload --eval_strategy steps --eval_steps 10 --split_dataset_ratio 0.05 --lr_scheduler_type warmup_stable_decay --lr_scheduler_kwargs "{ "num_decay_steps": 566, "num_stable_steps": 1225}" --dataloader_num_workers 4 --model_author swift --model_name swift-robot --log_level detail --logging_steps 1 --report_to wandb --dataset_shuffle False --train_dataloader_shuffle False --packing True
模型：qwen3-8b-base
loss截图：

<img width="3840" height="1678" alt="Image" src="https://github.com/user-attachments/assets/cef1e5c2-03d2-49f3-9ef4-745788794702" />

**Comment by chenchao9999** [comment]
我训练别的任务时也出现了，大概在最开始的700步的时候比较明显，数据量此时已经训到了27w条，但loss明显不是收敛的状态。因为我小模型能够训到更低的loss。

**Comment by chenchao9999** [comment]
有可能是学习率的问题，当时我设置的7e-5

**Comment by chenchao9999** [comment]
我把batch拉大后解决了，正常训练
